Documentation/ApiOverview/Backend/ContextualMenu.rst

```diff
@@ -312,3 +312,22 @@ If :yaml:`autoconfigure` is disabled you can manually register a context menu it
 by adding the tag :yaml:`backend.contextmenu.itemprovider`:
 
 ..  include:: /CodeSnippets/Tutorials/ContextMenu/ManualServicesYaml.rst.txt
+
+Migration of the JavaScript
```

Comment:
```suggestion
Migration of JavaScript
```

Hmm ... don't like the header, find it that better: "Binding this to context menu" -> it makes clearer what is the purpose and search results may get better.